jetty-core/jetty-util/src/main/java/org/eclipse/jetty/util/Scanner.java

```diff
@@ -195,7 +188,8 @@ public FileVisitResult preVisitDirectory(Path dir, BasicFileAttributes attrs) th
                 if (accepted)
                 {
                     scanInfoMap.put(dir, new MetaData(f.lastModified(), f.isDirectory() ? 0 : f.length()));
-                    if (LOG.isDebugEnabled()) LOG.debug("scan accepted dir {} mod={}", f, f.lastModified());
+                    if (LOG.isDebugEnabled())
```

Comment:
I can't see what is different here? Just white space?

Reply:
Just whitespace / following jetty format.